src/content/docs/en/guides/typescript.mdx

```diff
@@ -179,6 +179,56 @@ type MyAttributes = astroHTML.JSX.ImgHTMLAttributes;
 ```
 :::
 
+### getStaticPaths
```

Comment:
Not quite happy with this but two aims here:
1) Small style nit for `getStaticPaths()` being in backticks and including parens.
2) I’m trying to communicate that this is not the main `getStaticPaths()` docs in the heading — important for search results — while not making this too verbose in context on the page

```suggestion
### Infer `getStaticPaths()` types
```